app/views/transferComplete.scala.html

```diff
@@ -1,4 +1,4 @@
-@()(implicit messages: Messages, request: RequestHeader)
+@(consignmentReferenceData: ConsignmentReferenceData)(implicit messages: Messages, request: RequestHeader)
```

Comment:
Related to my comment about, whether this needs to a case class, instead of just passing a string directly.